docs/site/How-tos.md

```diff
@@ -1,26 +0,0 @@
----
```

Comment:
Is the deletion intentional?

Reply:
Yes, this is intentional. This page doesn't show up in our docs. In the recent change, if there's no `url` in the sidebar item, clicking on the link will only expand the tree.